providers/flagd/README.md

```diff
@@ -33,30 +34,16 @@ Flag configurations for evaluation are obtained via gRPC protocol using [sync pr
 Consider following example to create a `FlagdProvider` with in-process evaluations,
 
 ```go
-provider := flagd.NewProvider(
+provider, err := flagd.NewProvider(
         flagd.WithInProcessResolver(),
         flagd.WithHost("localhost"),
-        flagd.WithPort(8013))
+        flagd.WithPort(8013),
```

Comment:
hmm inprocess uses default port 8015 should we use this port here?

Reply:
i would stick with the defaults i think.... i case of doubt, we always check what the java provider is doing :D